packages/eslint-plugin-office-custom-functions/src/rules/__tests__/no-office-api-calls.test.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+exports.__esModule = true;
+var experimental_utils_1 = require("@typescript-eslint/experimental-utils");
+var resolve_from_1 = __importDefault(require("resolve-from"));
+var no_office_api_calls_1 = __importDefault(require("../no-office-api-calls"));
+var ruleTester = new experimental_utils_1.TSESLint.RuleTester({
+    parser: resolve_from_1["default"](require.resolve('typescript-eslint'), 'parser'),
+    parserOptions: { ecmaVersion: 2015 }
+});
+ruleTester.run('no-office-api-calls', no_office_api_calls_1["default"], {
+    valid: [
+        {
+            code: "\n                /**\n                 * Displays the current time once a second.\n                 * @customfunction\n                 * @param invocation Custom function handler\n                 */\n                export function clock(invocation: CustomFunctions.StreamingInvocation<string>): void {\n                const timer = setInterval(() => {\n                    const time = currentTime();\n                    invocation.setResult(time);\n                }, 1000);\n                \n                invocation.onCanceled = () => {\n                    clearInterval(timer);\n                };\n                }\n            "
```

Comment:
Can we put this code strings in a helper file as string variables and then import them to make the test code look cleaner?